packages/core/src/parser/Style.ne

```diff
@@ -36,7 +36,12 @@ const styleTypes: string[] =
 const lexer = moo.compile({
   ...basicSymbols,
   identifier: {
-    match: /[A-z_][A-Za-z_0-9]*/,
+    // Allow alpha, numbers, underscores, and dashes.  But an identified name:
+    //  - May not begin or end with a dash.
+    //  - May not begin with a number (ending with a number is ok)
+    //  - May be a single alpha character or underscore
+    //  - May begin or end with an underscore
+    match: /[A-Za-z_](?:[A-Za-z_0-9\-]*[A-Za-z_0-9])*/,
```

Comment:
In #427 we decided to disallow kebab case altogether. So I think we should remove dashes in the regex.

Reply:
It is a good point.  Based on Nimo's comment, we re-discussed this using #427 as context.  The desire to allow passthrough SVG properties has shifted thinking somewhat, yet a desire to allow "x1-x2" type syntax for subtraction remains strong vs. requiring a clarifying space.  Given the tradeoffs, we decided to think about alternatives such as bracketing identifiers containing dashes and sleep on it.

Reply:
Resolving this since we decided to postpone and not allow hyphens in this PR.